core/schema.py

```diff
@@ -1148,6 +1162,17 @@ def update_or_create_user(data, user):
     client_mutation_id = data.get("client_mutation_id", None)
     # client_mutation_label = data.get("client_mutation_label", None)
 
+    incoming_email = data.get('email')
+    current_user = InteractiveUser.objects.filter(user__id=data['uuid']).first()
+
+    current_email = current_user.email if current_user else None
+
+    if current_email != incoming_email:
+        if not incoming_email:
+            pass
+        elif check_user_unique_email(user_email=data['email']):
+            raise ValidationError(_("mutation.user_email_duplicated"))
```

Comment:
Key not present in module and Lokalise BE.

Reply:
Already added.